crates/services/compression/src/service.rs

```diff
@@ -649,4 +649,39 @@ mod tests {
             .unwrap();
         assert!(maybe_block.is_some());
     }
+
+    #[cfg(feature = "fault-proving")]
+    #[tokio::test]
+    async fn compression_service__merkleizes_registrations_per_table() {
```

Comment:
We're not testing any merkleization here, I suppose the name should be updated.

Reply:
addressed in e5aeb4b9c3